### PR TITLE
Update to 7.12.0

### PR DESCRIPTION
**Pull request by RhCheGithubBot**
Tracking changes for fixing compatibility with upstream 7.12.0-SNAPSHOT. This PR was created automatically by Jenkins from job devtools-rh-che-rh-che-compatibility-test-dev.rdu2c.fabric8.io

### CODE REVIEW

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
@ibuziuk It seems like the problem is not in tests this time. Workspace is RUNNING but never switch to IDE [1]. I've tried that manually with a user `kkanova-preview` and I can see those console logs [2]. I tried to refresh the page and got those console logs [3]. Tests are failing continuously on that so I would suppose it is some bug on hosted che.

[1] 
image

[2]
Screenshot from 2020-04-15 14-33-51

[3] 
Screenshot from 2020-04-15 14-34-41

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

File: dockerfiles/e2e-saas/docker-entrypoint.sh [hunk 1]
```diff
@@ -179,5 +192,16 @@ fi
...
+mkdir -p $ffmpeg_path
+nohup ffmpeg -y -video_size 1920x1080 -framerate 24 -f x11grab -i $DISPLAY.$SCREEN $ffmpeg_path/output.mp4 2> $ffmpeg_path/ffmpeg_err.txt > $ffmpeg_path/ffmpeg_std.txt & 
+ffmpeg_pid=$!
+set +x
+trap kill_ffmpeg 2 15
+
 echo "Running test suite: $TEST_SUITE"
 npm run $TEST_SUITE
+
+kill_ffmpeg
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/docker-entrypoint.sh:207]: no new line

**Comment by Katka92** [comment]
@ibuziuk The last three failures are related to this [1]. Do you know what to do with that - is that infra or che issue? I'm able to reproduce that manually, it happens really fast when the user creates a workspace and wants to start it.

[1] 
image

**Comment by ibuziuk** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
@ibuziuk The job is failing on a build step. Could you please take a look? Thanks

**Comment by ibuziuk** [comment]
[test]

**Pull request merged**